lib/Service/InvoiceService.php

```diff
@@ -221,11 +221,11 @@ public function upcoming($params = null, $opts = null)
      *
      * @throws \Stripe\Exception\ApiErrorException if the request fails
      *
-     * @return \Stripe\Invoice
+     * @return \Stripe\Collection
      */
     public function upcomingLines($params = null, $opts = null)
     {
-        return $this->request('get', '/v1/invoices/upcoming/lines', $params, $opts);
+        return $this->requestCollection('get', '/v1/invoices/upcoming/lines', $params, $opts);
```

Comment:
Is this expected?

Reply:
Yes. This will fix a bug in pagination on these list methods.